src/rtc.c

```diff
@@ -0,0 +1,518 @@
+/**
+ * @file rtc.c
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+
+#include "libdragon.h"
+
+/**
+ * @defgroup rtc Real-Time Clock Subsystem
+ * @ingroup libdragon
+ * @brief Joybus real-time clock interface.
+ *
+ * The Joybus real-time clock is an accessory that was only included on
+ * one official cartridge that was only available in Japan. The Joybus RTC
+ * is accessed through the serial interface (SI) similar to EEPROM.
+ * A real RTC uses a battery to power a clock that runs even when the N64
+ * is powered-off and maintains the date, time, and day of the week.
+ *
+ * To check if the real-time clock is available, call #joybus_rtc_is_present.
+ * If detected, it is a good idea to ensure the RTC control block is in a
+ * known-good state by calling the #joybus_rtc_run function before reading
+ * the current time with #joybus_rtc_read_time for the first time.
+ *
+ * The simplest method to change the time is by calling #joybus_rtc_set,
+ * which is a high-level convenience function that prepares the RTC and
+ * writes the time with the proper delays.
```

Comment:
I'd say this paragraph could be simplified since there's only one way now, and also details about delays are irrelevant in a general overview.

I'd mention timing, though. Maybe people won't expect a write to take many milliseconds